src/tools/perf/lib/libperf.c

```diff
@@ -1537,8 +1557,7 @@ static void* ucx_perf_thread_run_test(void* arg)
                 goto out;
             }
         }
-#pragma omp master
-        ucx_perf_test_reset(perf, params);
+        ucx_perf_test_prepare_new_run(perf, params);
```

Comment:
probably need to update single-threaded mode as well? to call ucx_perf_test_prepare_new_run instead of ucx_perf_test_reset

Reply:
Will do.